### ISSUE TITLE
"Store Closed" Component

### ISSUE THREAD

**Issue by paulchrisluke** [whynotearth/foodouken.com#16]
# "Store Closed" Page
![image](https://user-images.githubusercontent.com/5694308/78332779-e08a2600-75b2-11ea-9f0f-90afe012d590.png)
![image](https://user-images.githubusercontent.com/5694308/78331841-10d0c500-75b1-11ea-876a-c31a75c97d55.png)
This is the screen displayed as an overlay when someone is accessing the site of a restaurant currently closed. 

The "Closed" screen has a few parts: 
1. illustration
2. Title
3. Text

### 1. Illustration 
![image](https://user-images.githubusercontent.com/5694308/78332259-d9aee380-75b1-11ea-81aa-1def03f82b7b.png)

This illustration is 328 pixels by 262 pixels, centered 112 pixels from the top. Cloudinary link below:
https://res.cloudinary.com/whynotearth/image/upload/v1585896304/foodouken/Store_Closed_Illustration_mgecis.png

Inside the closed sign is also an image of the restaurant's logo
Bang Bang Bakery's Logo cloudinary link is here: 
https://res.cloudinary.com/whynotearth/image/upload/v1585896360/foodouken/bangbangbakerycafe/BangBang_Bakery_Logo_ezhmca.png

### 2. Title 
![image](https://user-images.githubusercontent.com/5694308/78332336-0bc04580-75b2-11ea-8cc2-0c8a5dadc9ae.png)
This title will state the name of the restaurant and display that it is closed. It is centered and is 40 pixels from the illustration and the text below

Mobile: 
Font: Mobile H1
Color: on Background 
Emphasis: High 

Desktop: 
Font: Desktop H2 
Color: On Background 
Emphasis: High

### 3. Text
![image](https://user-images.githubusercontent.com/5694308/78331256-d7e42080-75af-11ea-999d-bf12040c73d7.png)
The text will display their business hours. It is centered and 40 pixels below the title. 

Mobile: 
Font: Mobile Body 1
Color: On Background 
Emphasis: Medium 

Desktop: 
Font: Desktop Body 1
Color: On Background
Emphasis: Medium

# Design File
For design reference, please see files in [Figma](https://www.figma.com/file/Q671pbdFk3omco20xmxk5o/foodouken.com?node-id=1245%3A57965)

**Comment by GerardBalaoro** [comment]
@paulchrisluke In the API, the `custom.openingHours.days` property contains an array of objects `{ day: 0, ... }`. Would it be possible if the API could communicate the shortened names of the week such that it is `{ day: "mon" }`? If not, are we to use Monday as day 0 or Sunday?

I also think it would be a good idea if there is a `closed` or `isClosed` property in the API to determine if the store is accepting orders regardless of the day or time.

**Comment by paulchrisluke** [comment]
@GerardBalaoro good ideas, I can re-write that json any way we like. Can you give me an example of what you are looking for and I can get that changed for you?

**Comment by GerardBalaoro** [comment]
@paulchrisluke I was thinking for something like this.

```js
{
  id: 204
  title: "Bang Bang Bakery Cafe"
  closed: false
  ...
}
```

For the opening hours:
```js
{
  openingHours: {
    days: [
      {
        day: "mon",
        startTime: "0800",
        endTime: "1500"
      }
    ]
  }
}
```

**Comment by GerardBalaoro** [comment]
@paulchrisluke It would also be better if the logo of the store is provided by the API.

**Comment by GerardBalaoro** [comment]
The store name is displayed by default at the NavBar component which is not a child of the Home component in which the store page is loaded. I propose we instead put "Sorry, We're Closed".

![localhost_8080_](https://user-images.githubusercontent.com/13101267/78761209-e1f18f00-79b4-11ea-81f1-98993a292ba0.png)

The text containing store schedules are harder to implement since the API provides an array of days with start and end times. It would be easier to just list them out using a table (or cards)

**Comment by paulchrisluke** [comment]
Closing the closed store. We made some more decisions about how to handle store closed. Instead of not allowing the user to order at all, we handle this in the ordering form, allowing the user to order later. We will also be building a home index, which shows all resturaunts in a list. I anticipate we will show closed stores at the bottom and greyed out. This data structure will still be used for that. We may use this popup at some point, but can close for now.

Thanks for your contribution @GerardBalaoro , got our JSON structures whipped into shape!
